files/ko/web/javascript/guide/expressions_and_operators/index.html

```diff
@@ -11,454 +11,721 @@
+      <td><code>x &gt;&gt;= y</code></td>
+      <td><code>x = x &gt;&gt; y</code></td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/ko/docs/Web/JavaScript/Reference/Operators/Unsigned_right_shift_assignment"
+          >부호 없는 오른쪽 시프트 할당</a
+        >
+      </td>
+      <td><code>x &gt;&gt;&gt;= y</code></td>
+      <td><code>x = x &gt;&gt;&gt; y</code></td>
+    </tr>
+    <tr>
+      <td><a href="/ko/docs/Web/JavaScript/Reference/Operators/Bitwise_AND_assignment">비트 AND 할당</a></td>
+      <td><code>x &amp;= y</code></td>
+      <td><code>x = x &amp; y</code></td>
+    </tr>
+    <tr>
+      <td><a href="/ko/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR_assignment">비트 XOR 할당</a></td>
+      <td><code>x ^= y</code></td>
+      <td><code>x = x ^ y</code></td>
+    </tr>
+    <tr>
+      <td><a href="/ko/docs/Web/JavaScript/Reference/Operators/Bitwise_OR_assignment">비트 OR 할당</a></td>
+      <td><code>x |= y</code></td>
+      <td><code>x = x | y</code></td>
+    </tr>
+    <tr>
+      <td><a href="/ko/docs/Web/JavaScript/Reference/Operators/Logical_AND_assignment">논리 AND 할당</a></td>
+      <td><code>x &amp;&amp;= y</code></td>
+      <td><code>x &amp;&amp; (x = y)</code></td>
+    </tr>
+    <tr>
+      <td><a href="/ko/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment">논리 OR 할당</a></td>
+      <td><code>x ||= y</code></td>
+      <td><code>x || (x = y)</code></td>
+    </tr>
+    <tr>
+      <td><a href="/ko/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment">널 병합 할당</a></td>
+      <td><code>x ??= y</code></td>
+      <td><code>x ?? (x = y)</code></td>
+    </tr>
+  </tbody>
 </table>
 
-<h4 id="구조_분해">구조 분해</h4>
+<h4 id="return_value_and_chaining">반환 값과 체이닝</h4>
+
+<p>
+  <code>x = y</code>와 같은 할당 역시, 대부분의 표현식처럼 값을 반환합니다. 할당 연산자의 반환 값은 다른 할당 연산자나
+  콘솔 기록 등을 통해 가져올 수 있습니다.
+</p>
 
-<p>복잡한 할당 연산에서, <a href="/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment">구조 분해 할당</a> 구문은 배열이나 객체의 구조를 반영하여 배열이나 객체에서 데이터를 추출할 수 있게 해주는 JavaScript 표현식입니다.</p>
+<pre class="brush: js">
+const z = (x = y)  // const z = x = y와 같음
 
-<pre class="brush: js">var foo = ['one', 'two', 'three'];
+console.log(z)     // x = y 할당의 반환 값을 기록
+console.log(x = y) // 또는 반환 값을 직접 기록
+</pre>
 
-// 구조 분해를 활용하지 않은 경우
-var one   = foo[0];
-var two   = foo[1];
-var three = foo[2];
+<p>
+  반환 값은 위쪽 표의 "뜻" 열에서 등호(<code>=</code>)의 오른쪽 표현식 값과 동일합니다. 즉 <code>(x = y)</code>는
+  <code>y</code>를 반환하고, <code>(x += y)</code>는 <code>x + y</code>의 결과를, <code>(x **= y)</code>는
+  <code>x ** y</code>의 결과를 반환합니다.
+</p>
 
-// 구조 분해를 활용한 경우
-var [one, two, three] = foo;</pre>
+<p>
+  논리 할당인 <code>(x &amp;&amp;= y)</code>, <code>(x ||= y)</code>, <code>(x ??= y)</code>의 경우에는 할당 부분을
+  제외한 논리 연산의 결과를 반환합니다. 따라서 각각 <code>x &amp;&amp; y</code>, <code>x || y</code>,
+  <code>x ?? y</code>의 결과를 반환합니다.
+</p>
 
-<h3 id="비교_연산자">비교 연산자</h3>
+<p>논리 할당의 반환 값은 <b>할당 전</b>의 값을 사용한 논리 연산의 결과임에 주의하세요.</p>
```

Comment:
```suggestion
<p>반환 값은 <em>할당 전</em>의 값을 사용한 논리 연산의 결과임에 주의하세요.</p>
```
원문은 ` Note that the return values are always based on the operands’ values before the operation.` 인데요, 조금 더 비슷하게 바꾸어보았습니다.